### PR TITLE
Update 08.outlook.md

### LINKED ISSUE #68: Outlook section content
Drop thoughts here for anything that can be included in the outlook section

### PR DESCRIPTION
**Pull request by jaybee84**
### Purpose

closes #68 

#### Which areas should receive a particularly close look?

1. content: is there anything else we want to highlight
2. structure: currently the structure is a) recap, b) two broad concepts where we encourage emphasis with some examples

#### Is the pull request ready for review?

Yes

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [x] All terms that need to be added to glossary have been added to #50 
- [x] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

File: content/08.outlook.md
Comment (jaclyn-taroni) [inline content/08.outlook.md]: Citation? May be covered by `[TODO: add Refs]` below 😄

File: content/08.outlook.md
Comment (jaclyn-taroni) [inline content/08.outlook.md]: I am hesitant to talk about a literature search for "this is not a systematic review" reasons.

**Review by jaclyn-taroni — COMMENTED** [review]
👍  This content looks like it is headed in the right direction and is consistent with what we've discussed throughout putting the manuscript together! I will have more specific comments of a wordsmithing nature in the future, but I don't think that was the point of this review.

I left two comments when things jumped out at me, though.

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]: We didn't specifically interrogate this in this paper, I think, or am I misremembering?

Comment (allaway) [reply content/08.outlook.md]: e.g. I think we had this hypothesis but didn't follow this particular thread (primarily because we don't have non-laser resected cNF tumor expression data to explore this difference). I think if someone looked for this in the cited paper, they wouldn't find it...but again, could be misremembering.

Comment (jaybee84) [reply content/08.outlook.md]: You are correct... we had this hypothesis but did not interrogate it systematically in the paper. As I am re-reading this, I see that my phrasing suggests more than what I intended to say. Will rephrase :)

File: content/08.outlook.md
Comment (jaybee84) [reply content/08.outlook.md]: :)

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
Throughout this perspective, we have highlighted various challenges in applying ML methods to rare disease data as well as examples of approaches that address these challenges.
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
Scarcity of samples, while significant, is not the only roadblock towards application of ML in rare disease data.
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
The high dimensionality of modern data requires creative approaches, such as learning new representations of the data, to manage the curse of dimensionality.
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
It further requires leveraging prior knowledge and transfer learning methods to appropriately interpret data. 
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]: I'm not sure what the point is here - little confused!

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]: Just checking is this a TODO or a section header? (If a section header, suggest removing the period)

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md:19]: "While the integrative analysis in the study using transfer learning techniques was able to minimize technique related signals" 

I don't think that we really tested this, hesitant to make this claim.

Comment (allaway) [reply content/08.outlook.md:19]: After we merge, I can tackle this section - I have some ideas but a little too much rewriting to do through Github PR suggestions :)

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]: Can we cite Sci Data paper instead? Too much risk of this URL going dead. 
```suggestion
A few such collaborations are beginning to show promise in generating valuable datasets for future use.[@doi:10.1038/s41597-020-0508-5]

```

Comment (allaway) [reply content/08.outlook.md]: I do wonder if this sentence is laying on the NF focus a bit too much, but leave that to other reviewers to determine (i'm biased)

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
With the ubiquity of sequencing platforms, genomic data has been, relatively speaking, easy to gather for rare disease patients.[@doi:10.1038/nrg3555; @doi:10.1038/nrg.2017.116; @doi:10.1056/NEJMra1711801]
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
An important next step is to develop comprehensive comprehensive genomics-driven genotype-phenotype databases that can fuel interpretation of features extracted using ML methods.
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md:27]: I think @cgreene could review and expand on this in his round-robin if it seems necessary!

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
The majority of ML methods for rare disease that we have investigated are applied to classification tasks. 
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
Conversely, we've found few examples of methodologies that interrogate biological mechanisms of rare diseases. 
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
This is likely a consequence of a dearth of methods that can tolerate the constraints imposed by rare disease research such as phenotypic heterogeneity and limited data.
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
An intentional push towards developing methods or analytical workflows that address this will be critical to apply machine learning approaches to rare disease data.
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
One way to increase trust in a developed model is by helping users understand the behavior of the developed model through providing explanations regarding why a certain model made certain predictions.[@https://www.aclweb.org/anthology/N16-3020.pdf]
```

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
Finally, methods that can reliably integrate disparate datasets will always remain a need of rare diseases.
```

Comment (allaway) [reply content/08.outlook.md]: why?

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md:44]: This seems semi-redundant with above sentence

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md:45]: This is a complicated topic to introduce as second to last sentence, I suggest removing or moving elsewhere

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
Overall, we speculate that this an important burgeoning area of research, and we are optimistic about the future of applying machine learning approaches to rare disease.
```

**Review by allaway — CHANGES_REQUESTED** [review]
Looks great! I made some suggestions.

File: content/08.outlook.md
Comment (jaybee84) [reply content/08.outlook.md]: This was to recap the "managing model complexity" section of the article. 
This sentence hopes to remind the reader of fundamental techniques like bootstrap and regularization ... but as is evident may need rephrasing :)

File: content/08.outlook.md
Comment (jaybee84) [inline content/08.outlook.md]:
```suggestion
_Emphasis on not just "more n" but "more meaningful n"_
```

File: content/08.outlook.md
Comment (jaybee84) [reply content/08.outlook.md:19]: This is based on some of the discussion during writing of the earlier manuscript that since the latent variable generation finds patterns that are consistent among datasets that have divergent technical differences in them,  the result is enrichment of LVs that are biologically relevant (since the technical differences would presumably be orthogonal in different datasets). 
But maybe I am misremembering... Tagging @cgreene and @jaclyn-taroni for more insight

Comment (jaybee84) [reply content/08.outlook.md:19]: Also.. happy to have you tackle this section after merging :)

File: content/08.outlook.md
Comment (jaybee84) [reply content/08.outlook.md]: maybe we can try to find other rare disease dataset releases to cite here

File: content/08.outlook.md
Comment (jaybee84) [reply content/08.outlook.md]: My thoughts are as follows:
1. Since rare disease patients will be sparse by the virtue of rarity of disease...Even if we can ensure that every NF patient sample in US is collected in a standardized manner, patients in UK will be processed differently... but may need to be integrated to have a more substantial dataset.
2. There will always be temporal differences in sample collection and processing, due to the nature of clinical practice. In diseases that are common, samples with significant temporal differences can be disregarded from an analysis, but that luxury is not available to the rare disease field.

File: content/08.outlook.md
Comment (jaybee84) [reply content/08.outlook.md:44]: The above sentence was referring to datasets that are of the same datatype e.g. RNAseq. This is referring to datasets using different datatypes e.g. RNAseq + methylation

This segues to the next "anchor" section.

File: content/08.outlook.md
Comment (jaybee84) [reply content/08.outlook.md:45]: I agree that it is a complicated topic, but it would be great to have this in the outlook. Happy to move it towards the beginning, or delve a bit deeper into this if everyone is on board.

This is based on a discussion with @jguinney, and I thought it was good food for thought (both for us and the readers) :)

File: content/08.outlook.md
Comment (allaway) [inline content/08.outlook.md]:
```suggestion
Additionally, anyone applying machine learning methods on rare disease data should use techniques that increase confidence (such as bootstrapping) and penalize complexity of the resultant models (such as regularization) to enhance the generalizability of their work. 
```

File: content/08.outlook.md
Comment (allaway) [reply content/08.outlook.md]: Made a suggested change to reflect this/improve clarity!

File: content/08.outlook.md
Comment (allaway) [reply content/08.outlook.md:19]: Alrighty, filed as #143!

**Review by allaway — APPROVED** [review]
Left one additional suggested change!

**Pull request merged**
